run.sh

```diff
@@ -15,7 +15,8 @@ fi
 
 exec 2>&1
 
-java -jar /opt/exhibitor/exhibitor.jar \
+java -javaagent:/agents/appdynamics-jvm/javaagent.jar -Dappdynamics.agent.uniqueHostId=$(cat /agents/appdynamics-jvm/uniqueHostId) \
```

Comment:
It looks like you forgot to update Dockerfile. Can't find any mentions of appdynamics there.